iXBRLViewerPlugin/viewer/src/img/workiva.svg

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Need to add the copyright info to this file as well.

Reply:
Done in 5148a99.